src/v2/guide/team.md

```diff
@@ -1379,19 +1382,25 @@ order: 803
         if (!vm.userPosition) return vuers
         var vuersWithDistances = vuers.map(function (vuer) {
           var cityCoords = cityCoordsFor[vuer.city]
+          if (cityCoords) {
+            return Object.assign({}, vuer, {
+              distanceInKm: getDistanceFromLatLonInKm(
+                vm.userPosition.coords.latitude,
+                vm.userPosition.coords.longitude,
+                cityCoords[0],
+                cityCoords[1]
+              )
+            })
+          }
           return Object.assign({}, vuer, {
-            distanceInKm: getDistanceFromLatLonInKm(
-              vm.userPosition.coords.latitude,
-              vm.userPosition.coords.longitude,
-              cityCoords[0],
-              cityCoords[1]
-            )
+            distanceInKm: null
           })
         })
         vuersWithDistances.sort(function (a, b) {
           return (
-            a.distanceInKm -
-            b.distanceInKm
+            a.distanceInKm === null) - (b.distanceInKm === null) ||
```

Comment:
Are you sure `(a.distanceInKm === null) - (b.distanceInKm === null)` is the expected statement?

Reply:
Definitely :) I'll update it.

Reply:
Changed the condition here: https://github.com/vuejs/vuejs.org/pull/2109/commits/db5f3156c6e256bd5cc0bee2512c17b6073e2f89